uvloop/sslproto.pyx

```diff
@@ -277,13 +28,13 @@ class _SSLProtocolTransport(aio_FlowControlMixin, aio_Transport):
 
     def get_extra_info(self, name, default=None):
         """Get optional transport information."""
-        return self._ssl_protocol._get_extra_info(name, default)
+        return (<SSLProtocol>self._ssl_protocol)._get_extra_info(name, default)
```

Comment:
Would it be a good idea to make this a `cdef class`? We don't need too many methods from `asyncio.Transport` (and working `isinstance(ssl_proto_transport, asyncio.Transport)` is not a requirement.

If we make this an extension class you won't need `<SSLProtocol>` casts.

Reply:
Yeah I was thinking about that too, but didn't go that way because I found no `cdef` methods in transport, and it was checked somewhere that `FlowControlMixin` must be a parent class. Let me check and get back to you. If possible, I prefer `cdef class` too with less dict lookups. 😈

Reply:
Got it, I'll take a quick look and include it if it's simple